targets/unity-v2/Testing/Tests/Client/PlayFabAuthServiceTests.cs

```diff
@@ -14,7 +14,60 @@ public override void Tick(UUnitTestContext testContext)
     private PlayFabAuthService _emailAuthService;
 
     [UUnitTest]
-    public void EmailPasswordLoginTest(UUnitTestContext testContext)
+    public void InvalidEmailPasswordLogin(UUnitTestContext testContext)
+    {
+        const string password = "1";
+        const string username = "1";
+        const string email = "LoginTest.com";
+        
+        var authService = new PlayFabAuthService();
+        authService.Password = password;
+        authService.Username = username;
+        authService.Email = email;
+
+        authService.OnLoginSuccess += success => testContext.Fail("Login fail expected.");
```

Comment:
nitpick: i prefer seeing parenthese around the parameters before the => to visually note this as a function of a sort.